app/controllers/concerns/sign_in/sso_authorizable.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module SignIn
```

Comment:
Is there an advantage for this to be included in the sign in controller? I was only able to find one function from sign in controller that this uses, and I don't think it even needs that function.

The other advantage (although this is completely a secondary advantage), is that we don't have to have these specs in `sign_in_controller_specs.rb`, as that spec files is already way too big

Reply:
I just split in out into a concern since the `SignInController` is massive. So basically instead of directly adding the `authorize_sso` method to `SignInController` we can just include this module instead. 

We could also put all this logic in a service object and then the `authorize_sso` method in `SignInController` could be skinny. Someday we should clean up the rest of `SignInController`  😅

Reply:
What about having it a completely different controller?